pkg/config/config.go

```diff
@@ -53,10 +53,16 @@ type Config struct {
 // Resource contains resources to watch
 type Resource struct {
 	Name       string
-	Namespaces []string
+	Namespaces Namespaces
 	Events     []EventType
 }
 
+// Namespaces contains namespaces to include and ignore
```

Comment:
Please add documentation about Include, Ignore as well here.
Can be added from above yaml file ?
Eg. meaning of `all` etc.

Reply:
Added them